pkgs/applications/misc/terminus/default.nix

```diff
@@ -28,16 +88,22 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/bin"
     mv opt "$out/"
     ln -s "$out/opt/Terminus/terminus" "$out/bin/terminus"
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" --set-rpath "${libPath}:\$ORIGIN" "$out/opt/Terminus/terminus"
+
+    patchelf --set-interpreter ${stdenv.cc.bintools.dynamicLinker} \
+        "$out/opt/Terminus/terminus"
+
+    wrapProgram $out/bin/terminus \
+        "''${gappsWrapperArgs[@]}" \
+        --prefix XDG_DATA_DIRS : "${gtk3}/share/gsettings-schemas/${gtk3.name}/" \
+        --prefix LD_LIBRARY_PATH : ${libPath}
```

Comment:
this really shouldn't be done at all (we should use rpath like before).
Have you tried using `autoPatchelfHook`? It patches things automatically and will check if a dependency that is needed is missing in `buildInputs`.

Reply:
Thanks for your help, but with the proposed changes I'm always getting some kind of error. I'm not that familiar with those functions, and my try-and-error approach is not effective. I copied that approach from the `discord` derivation. Unfortunately, I don't have time to spend on this, feel free to close this PR if it makes sense to you.

Reply:
Totally understandable if you don't want to spend more time on a package you don't use anymore. I believe you could just leave this open for its new maintainer

Reply:
Then, let's do that